posthog/temporal/data_imports/external_data_job.py

```diff
@@ -114,6 +116,14 @@ def update_external_data_job_model(inputs: UpdateExternalDataJobStatusInputs) ->
 
     close_old_connections()
 
+    rows_tracked = get_rows(inputs.team_id, inputs.schema_id)
+    if rows_tracked > 0 and inputs.status == ExternalDataJob.Status.COMPLETED:
+        msg = f"Rows tracked is greater than 0 on a COMPLETED job. rows_tracked={rows_tracked}"
+        logger.debug(msg)
+        capture_exception(Exception(msg))
+
+    finish_row_tracking(inputs.team_id, inputs.schema_id)
```

Comment:
I presume this would always get called in case of any failure? (Just double checking because this is probably the main thing that could go wrong since it could prevent future syncs if we think they're syncing more rows than they are)

Reply:
Yeah, we always wanna be calling this